meetings/2021-11-24.md

```diff
@@ -0,0 +1,82 @@
+* Pavlik: I have a question on auxiliary resources. I understand that only aux resources have a control on them. Is there a case on controlling an 
+* If we say there is a special access mode that refers to an auxiliary resource. It sounds to me like this approach is not using ACR on auxiliary resource. You would still need that path.
+* Matthieu: Are you asking if we have a way currently in Solid to define ACRs on ACRs?
+* Pavlik: Say you have controlRead on resource A, then you have a custom access mode to target the ACR. It doesn't apply to resource A it applies to the ACR of resource A.
+* Matthieu: My short answer is that it is not up to ACP to define the mechanism you are proposing. ACP has a simple extension point, which is access modes. Those access modes are not defined by ACP but by the protocol.  The Solid Protocol says that in order to be able to edit an auxiliary resource you need an access mode acl:Control. So solid says you need ACL control to edit the auxiliary resource.
+* Barath: Can you be in a situation where your permissions would allow you to delete a resource but not its auxiliary control resource?
+* Matthieu: First, That is in my opinion something that the SOlid Protocol should define precisely. Moreover, I think in order to delete a resource as defined in the Solid Protocol currently you need acl:Control on both the resource and its container. So short answer, it should not be possible.
+* Henry: There is actually quite a few other good things that can happen from this change. Let me think... I think what Matthieu is saying is that one could open another use case for defining ACLs on ACLs rather than control. I think that someone was arguing giving access control to sub-parts of a graph. Even though it wasn't mentionned.
+* Matthieu: I could have a go at describing this. This change highlights the difference between a Solid Resource which is a document and resources as defined by RDF (whether just part of a representation, dereferenceable via 200 or 303)...
+* Henry: From the client's perspective, it can follow links and arrive on an ACR. From a client's perspective there is no reason why you should differentiate between an ACL and another resource. This is why you should be able to define that every single resource on the web has an access control rule. So we're just defining an ontology that defines that.
+* Elf: Why couldn't we just use the Read control mode instead of ControlRead.
+* Henry: It's a legacy.
+* Elf: But at this time we just use control read to say it applies to the auxiliary resource. If we use control read we don't target the acr, we target the resource that ACR is auxiliary of. Do you see the difference?
+* Henry: That's why control is weird. You only need read and write. The client needs a way to define that. And it is part of another proposal, not the one Matthieu is putting forward. All you would have to do is follow the link header and it could be the same resource.
```

Comment:
```suggestion
* Elf: But, at this time, we just use `control read` to say it applies to the auxiliary resource. If we use `control read`, we don't target the acr, we target the resource that  the ACR is auxiliary of. Do you see the difference?
* Henry: That's why `control` is weird. You only need `read` and `write`. The client needs a way to define that. And it is part of another proposal, not the one Matthieu is putting forward. All you would have to do is follow the `link` header, and it could be the same resource.
```